.gitignore

```diff
@@ -42,3 +42,4 @@ master/setuptools_trial*
 master/buildbot/www/static/js/default/
 sandbox
 .idea/
+/.venv/
```

Comment:
Well I don't want the pattern to match `.venv/` folders in subdirectories. Only at the root of the repo.

I don't have a strong opinion about this though. If you want, I can change it :)